spec/openapi.yaml

```diff
@@ -624,137 +611,186 @@ components:
+                  example: "15.79"
+                filament used [mm]:
+                  type: string
+                  example: "5170.06"
+                filament_type:
+                  type: string
+                  example: "PLA"
+                fill_density:
+                  type: string
+                  description: Percent
+                  example: "20%"
+                initial_exposure_time:
+                  type: string
+                  description: Seconds
+                  example: "5"
+                layer_height:
+                  type: string
+                  description: Milimeters
+                  example: "0.3"
+                material_name:
+                  type: string
+                  example: "PLA Sandstorm Orange"
+                exposure_time:
+                  type: string
+                  description: Seconds
+                  example: "2"
+                max_exposure_time:
+                  type: string
+                  description: Seconds
+                  example: "3"
+                max_initial_exposure_time:
+                  type: string
+                  description: Seconds
+                  example: "3"
+                min_exposure_time:
+                  type: string
+                  description: Seconds
+                  example: "1"
+                min_initial_exposure_time:
+                  type: string
+                  description: Seconds
+                  example: "1"
+                nozzle_diameter:
+                  type: string
+                  description: Milimeters
+                  example: "0.4"
+                print_time:
+                  type: integer
+                  description: Seconds
+                  example: "2562"
+                printer_model:
+                  type: string
+                  enum: [ "MK3", "MK3S", "MINI" ]
+                support_material:
+                  type: string
+                  example: "PLA"
+                ironing:
+                  type: string
+                  example: "0"
+                required_resin_ml:
+                  type: string
+                  example: "123.4"
+                profile:
+                  type: string
+                  example: "ultra_fast"
+
+    FirmwareFileInfo:
+      allOf:
+        - $ref: '#/components/schemas/FileTreeObject'
+        - type: object
+          properties:
+            meta:
+              type: object
+              properties:
+                version:
+                  type: string
+                  description: Firmware version in text format
+                printer_type:
+                  type: integer
+                printer_version:
+                  type: integer
+
+    OtherFileInfo:
+      $ref: '#/components/schemas/FileTreeObject'
+
+    FolderInfo:
+      allOf:
+        - $ref: '#/components/schemas/FileTreeObject'
+        - type: object
+          properties:
+            file_count:
```

Comment:
I'm not sure about that information on Buddy FW. Please check it with @vorner.

Reply:
Well, considering we have the array with children, isn't this a bit redundant? But if you really need us to count them, we can do so, but will send the field _after_ the `children` array 😈